src/python/bot/tasks/train_gradientfuzz_task.py

```diff
@@ -13,9 +13,308 @@
 # limitations under the License.
 """GradientFuzz training task."""
 
+from builtins import str
+
+import glob
+import os
+import shutil
+import sys
+
+from bot.fuzzers.ml.gradientfuzz import constants
+from bot.fuzzers.ml.gradientfuzz import run_constants
+from build_management import build_manager
+from fuzzing import corpus_manager
+from google_cloud_utils import storage
 from metrics import logs
+from system import archive
+from system import environment
+from system import new_process
+from system import shell
+
+# Model script directory.
+GRADIENTFUZZ_SCRIPTS_DIR = os.path.join(
+    os.path.dirname(os.path.dirname(__file__)), 'fuzzers', 'ml', 'gradientfuzz')
+
+
+def get_corpus(corpus_directory, fuzzer_name):
```

Comment:
This looks very similar to the one in train_rnn_generator_task.py. Can we factor that out into a module that can be reused by both? Something like ml_train_utils.py or similar. 

Same with many other functions in this file.

Reply:
Can do -- it'll help with other integrations as well!